src/vivarium_gates_mncnh/constants/data_values.py

```diff
@@ -320,6 +320,11 @@ class __DeliveryFacilityTypes(NamedTuple):
         FACILITY_CHOICE.P_BEmONC: 0.340528,
     },
 }
+PROPORTION_IN_FACILITY_DELIVERIES = {
+    "Ethiopia": 0.492568,
+    "Nigeria": 0.520097,
+    "Pakistan": 0.771766,
+}
```

Comment:
Is this something that would be useful to have as an actual artifact key? It is covariate_id=51 which would make it easier to generalize to new locations if we ever do that.

Reply:
Oh thanks for bringing this up... I believe that we should be using the mean_value of the covariate data for all draws (no draw level variation) because that's what @NathanielBlairStahn used for the input of his delivery facility model

Reply:
I just updated this PR to reflect that :) https://github.com/ihmeuw/vivarium_research/pull/1765